### PR TITLE
add Zendesk link in the Validator Page + add a page for switching testnet

### PR DESCRIPTION
**Pull request by antoineherzog**

### CODE REVIEW

File: docs/iov-name-service/validator/01-setup.md [hunk 1]
```diff
@@ -212,4 +207,4 @@ IMAGE_TM_OPTS="\
...
 
-Once your sentry nodes and validator are sync'ed then the final step to becoming a validator is to submit your validator's pub_key to IOV.  **On your validator node**, execute `curl --silent --fail http://localhost:16657/status | jq -r .result.validator_info.pub_key.value` and reply with the resulting 44 character pub_key to the ticket that was issued to you when you applied for the validator program.  (There's no `create-validator` command like in Cosmos; validators are added via governance, which is just IOV on the testnet, for the moment.)
+Once your sentry nodes and validator are sync'ed then the final step to becoming a validator is to submit your validator's pub_key to IOV.  **On your validator node**, execute `curl --silent --fail http://localhost:16657/status | jq -r .result.validator_info.pub_key.value` and reply with the resulting 44 character pub_key to the ticket that was issued to you when you applied for [the validator program here](https://support.iov.one/hc/en-us/requests/new?ticket_form_id=360000417771).  (There's no `create-validator` command like in Cosmos; validators are added via governance, which is just IOV on the testnet, for the moment.)
```
Comment (orkunkl) [inline docs/iov-name-service/validator/01-setup.md]: `sync`ed`? I have never seen this work used like this. Maybe I can learn something new today :)

Comment (orkunkl) [inline docs/iov-name-service/validator/01-setup.md]: Also maybe `[the validator program here](https://support.iov.one/hc/en-us/requests/new?ticket_form_id=360000417771)` -> `[the validator program here](https://support.iov.one/hc/en-us/requests/new?ticket_form_id=360000417771 "Validator Program")`

File: docs/iov-name-service/validator/01-setup.md [hunk 2]
```diff
@@ -1,21 +1,16 @@
 ---
 id: setup
-title: Validator setup
-sidebar_label: Setup
+title: Setup a Validator Node
+sidebar_label: Setup a Validator Node
```
Comment (davepuchyr) [inline docs/iov-name-service/validator/01-setup.md]: I chose single nouns for the side_bar label and would like to maintain the parallelism with them.  Swithcing from nouns to phrases breaks the parallelism.

File: docs/iov-name-service/validator/01-setup.md [hunk 3]
```diff
@@ -1,21 +1,16 @@
...
 ---
 
 > IOV does not recommend using docker in production; however, it can be used to start building your validator's quality score on our testnet if you so choose.
 
 This document is not for beginners.  It assumes that you know how to setup a sentry node architecture for Tendermint nodes.
 
-> Hint: When upgrading from an old testnet to a new one, you can maintain your node id and validator pub_key across testnets by doing the following before performing the upgrade:
+## Apply to the validator program
```
Comment (davepuchyr) [inline docs/iov-name-service/validator/01-setup.md]: This should be at the top of the page, IMHO.

File: docs/iov-name-service/validator/01-setup.md [hunk 4]
```diff
@@ -1,21 +1,16 @@
...
 
-```sh
-su - iov
-set -o allexport ; source /etc/systemd/system/iovns.env ; set +o allexport # pick-up env vars
-cp -av ${DIR_WORK}/config/*_key.json ~
-exit
-```
+Please [register your information on this form](https://support.iov.one/hc/en-us/requests/new?ticket_form_id=360000417771), this is needed for the IOV Validator to review your information and to open a channel of communication with us. At the end of the article, you will need to use this channel of communication to give us your pub-key so we can upgrade your full-node to a validator.
```
Comment (davepuchyr) [inline docs/iov-name-service/validator/01-setup.md]: The link should open a new window.

File: docs/iov-name-service/validator/04-upgrade.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (davepuchyr) [inline docs/iov-name-service/validator/04-upgrade.md]: This step is too vital to not be included in the Setup page.  It's a huge pain for validators and us to deal with a new pub_key on every upgrade.

File: docs/iov-name-service/validator/01-setup.md (hunk 1 above)
Comment (davepuchyr) [inline docs/iov-name-service/validator/01-setup.md]: Open link in new window.

File: docs/iov-name-service/validator/01-setup.md (hunk 4 above)
Comment (davepuchyr) [inline docs/iov-name-service/validator/01-setup.md]: "pub-key" should be "pub_key"

File: docs/iov-name-service/validator/01-setup.md (hunk 1 above)
Comment (davepuchyr) [reply docs/iov-name-service/validator/01-setup.md]: It's short for "synchronized". :)

File: docs/iov-name-service/validator/01-setup.md (hunk 2 above)
Comment (antoineherzog) [reply docs/iov-name-service/validator/01-setup.md]: what is the rationnal of using nouns only? Cosmos SDK Documentation use sometimes verbs and nouns

File: docs/iov-name-service/validator/04-upgrade.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+id: upgrade
+title: Switching Testnet
+sidebar_label: Switching Testnet
```
Comment (antoineherzog) [inline docs/iov-name-service/validator/04-upgrade.md]: I created an addtionnal page about switching testnet so user is not overload with information not needed what he is starting to setup a full node. @davepuchyr

**Pull request merged**